src/VisualStudio/Core/Impl/Options/CodeStyleNoticeTextBlock.xaml

```diff
@@ -10,7 +10,11 @@
     d:DesignHeight="450" d:DesignWidth="800">
     <Run Text="{x:Static local:CodeStyleNoticeTextBlock.CodeStylePageHeader}"/>
     <Run Text=" "/>
-    <Hyperlink RequestNavigate="LearnMoreHyperlink_RequestNavigate" NavigateUri="{x:Static local:CodeStyleNoticeTextBlock.CodeStylePageHeaderLearnMoreUri}">
+    <Hyperlink 
+        x:Name="learnMoreHyperlink"
```

Comment:
probably don't need 'x:Name' here unless we need to address in code, which doesn't look like the case.